### PR TITLE
10197 move all incubator modules of elico8 to public GitHub website beian

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

**Comment by elicoidal** [comment]
This version of the module is too specific. I have asked Lixia to improve and add 2 certificates with no reference to beian etc. so that It would be generic (such as certificate1, certificate2)
I am not sure but I think this is half version we have for our website 
@seb-elico did we finally make the development based on my requirement?

**Comment by Rona111** [comment]
cc @elicoidal @duanyp1991 @siyuancn

File: website_certificate/README.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+ * have basic modules installed (website, base)
+
+Usage
+=====
+
+Configure the company certificate information in website setting, then you can see the certificate information display at the bottom right of website.
+
+
+For further information, please visit:
```
Comment (elicoidal) [inline website_certificate/README.rst]: remove forum information

File: website_certificate/README.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+ * https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
```
Comment (elicoidal) [inline website_certificate/README.rst]: remove if empty

File: website_certificate/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico Corp (www.elico-corp.com).
```
Comment (elicoidal) [inline website_certificate/__openerp__.py]: 2016

**Comment by elicoidal** [comment]
small details. other than that LGTM

**Comment by Rona111** [comment]
cc @elicoidal @duanyp1991 @siyuancn

**Comment by elicoidal** [comment]
LGTM

File: website_certificate/certificate_view.xml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    <data>
+        <record id="view_website_config_settings" model="ir.ui.view">
+            <field name="model">website.config.settings</field>
+            <field name="name">website.config.settings.form.inherit</field>
+            <field name="inherit_id" ref="website.view_website_config_settings"/>
+            <field name="arch" type="xml">
+                <div name="social_googleplus" position="after">
+                    <label for="certificate_number1"/>
+                    <div class="oe_inline">
+                        <field name="certificate_number1" placeholder="沪ICP备12005091号"/>
```
Comment (siyuancn) [inline website_certificate/certificate_view.xml:12]: why Chinese here?

File: website_certificate/certificate_view.xml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+                        <field name="certificate_url1" placeholder="http://www.sgs.gov.cn/lz/licenseLink.do"/>
+                    </div>
+                    <label for="certificate_logo1"/>
+                    <div class="oe_inline">
+                        <field name="certificate_logo1" widget="image" class="
+                            oe_avatar oe_left"/>
+                    </div>
+                    <label for="certificate_number2"/>
+                    <div class="oe_inline">
+                        <field name="certificate_number2" placeholder="OCA沪ICP备12005091号"/>
```
Comment (siyuancn) [inline website_certificate/certificate_view.xml:25]: same as above

File: website_certificate/certificate_view.xml (hunk 4 above)
Comment (elicoidal) [inline website_certificate/certificate_view.xml:12]: this is originally for the ICP. I think it doesnot hurt (second one could be for Comodo example)

**Comment by siyuancn** [comment]
LGTM

**Comment by elicoidal** [comment]
by the way this one has no UT. for the 9.0 version, we should have UT

**Pull request merged**